boltdesignsystem/bolt
# DS-491 Portal contact cards

## Jira

https://pegadigitalit.atlassian.net/browse/DS-491

## Summary

Adds `status.replies_label`, `custom_menu`, and `subheadline` props for the Listing Teaser.

## Details

1. Doc page added to demo how contact list can be created using Listing Teasers.
2. `custom_menu` prop added to handle additional use cases other than share menu and action menu.
3. `subheadline` prop added to create a slot for secondary headlines.
4. `status.replies_label` prop added to support custom label text if necessary.

## How to test

Run the branch locally and check the contact-list doc and status-and-action doc under Listing Teaser.
